cdc/model/protocol.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"fmt"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/ticdc/pkg/p2p"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+// This file contains a communication protocol between the Owner and the Processor.
```

Comment:
As a communication protocol, does it need a versioning mechanism?

Reply:
added